index.js

```diff
@@ -25,6 +25,18 @@ const jwt = require("jsonwebtoken");
 const pubsub = new PubSub();
 const http = require("http");
 
+const rateLimit = require("express-rate-limit");
+const xss  = require("xss-clean");
+const helmet = require("helmet");
+const mongoSanitize = require("express-mongo-sanitize");
+
+const apiLimiter = rateLimit({
+  windowMs: 60 * 60 * 1000, // 15 minutes
+  max: 500,// this can be edited in between
+  message:
+  "Too many accounts created from this IP, please try again after an hour"
```

Comment:
Could you please change this to "Too many requests from this IP, please try again after an hour"?